templates/wsl/index.html

```diff
@@ -134,163 +111,137 @@ <h3 class="p-matrix__title p-heading--4">Kubernetes</h3>
-          </p>
-        </div>
-      </li>
-    </ol>
-  </div>
+  <div class="row">
+    <div class="col-3">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/ccd68f67-CI_CD-white.svg",
+        alt="",
+        width="152",
+        height="64",
+        hi_def=True,
+        loading="lazy",
+        attrs={"style": "padding-bottom:1rem;padding-top:16px;"}
+        ) | safe
+      }}
+      <h4>Cross Platform Development</h4>
+      <p>Create and test your CI/CD pipelines locally on an Ubuntu WSL instance. When ready, publish to a cloud production environment running Ubuntu VMs.</p>
+      <p><a class="p-link--external p-link--inverted" href="https://youtu.be/DmfuJzX6vJQ">Discover how to easily access “Kubernetes on Windows with WSL2” in our webinar</a></p>
+    </div>
+    <div class="col-3">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/72c4ec88-Security-wht-outline.svg",
+        alt="",
+        width="70",
+        height="80",
+        hi_def=True,
+        loading="lazy",
+        attrs={"style": "padding-bottom:1rem"}
+        ) | safe
+      }}
+      <h4>Security</h4>
+      <p>Achieve the same first-class, out-of-the-box, compliant security
+      that is synonymous with Ubuntu.  With long-term support releases,  you'll have five years of security patches and updates.</p>
+      <p><a class="p-link--inverted" href="/about/release-cycle">Read more about the Ubuntu lifecycle and release cadence&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-3">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/274e1895-laptop-user-white.svg",
+        alt="",
+        width="80",
+        height="80",
+        hi_def=True,
+        loading="lazy",
+        attrs={"style": "padding-bottom:1rem"}
+        ) | safe
+      }}
+      <h4>Manage IT Infrastructure</h4>
+      <p>From the same workstation, manage mixed Linux and Windows infrastructure both on-prem and across public clouds.</p>
+      <p><a class="p-link--inverted" href="https://youtu.be/50bokFFOvtA">Learn from our community members and experts at the WSLConf&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-3">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/6f30f9a5-Linux-laptop.svg",
+        alt="",
+        width="147",
+        height="80",
+        hi_def=True,
+        loading="lazy",
+        attrs={"style": "padding-bottom:1rem"}
+        ) | safe
+      }}
+      <h4>Enterprise support</h4>
+      <p>Ubuntu is certified on WSL through close collaboration with Microsoft. Enterprise support is provided for Ubuntu from Azure to Windows workstations creating a seamless operating environment.</p>
+      <p><a class="p-button js-invoke-modal" data-testid="interactive-form-link" class="p-button" href="https://ubuntu.com/wsl/contact-us">Get in touch</a></p>
+    </div>            
+  </div>  
 </section>
 
 <section class="p-strip is-deep">
```

Comment:
This section "Getting Started with WSL" doesn't seem to be updated yet?

Reply:
This is being done in another PR here: https://github.com/canonical-web-and-design/ubuntu.com/pull/11346